console/data/docs/02-configuration.md

```diff
@@ -112,7 +112,20 @@ Without configuration, *Akvorado* will listen for incoming
 Netflow/IPFIX and sFlow flows on a random port (check the logs to know
 which one).
 
-### BMP
+### BMP/RIS Lookup
+
+There are two BMP/RIS implementations available:
+The default is the internal BMP component, intended to directly terminate BMP Sessions from routers.
+Alternatively, BioRIS is available, intended to lookup routing information from existing [bio-rd RIS](https://github.com/bio-routing/bio-rd/tree/master/cmd/ris) instances.
+
+You can choose by setting the `ris-provider`:
+```yaml
+inlet:
+  core:
+    ris-provider: bioris
```

Comment:
How would you folks feel about using `BioRIS` here as is used internally?

Reply:
I'd say the current, lower case bioris variant fits best to the existing scheme: https://github.com/akvorado/akvorado/blob/main/inlet/core/config.go#L72